pandas/_libs/tslibs/parsing.pyx

```diff
@@ -0,0 +1,688 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# cython: profile=False
+# cython: linetrace=False
+# distutils: define_macros=CYTHON_TRACE=0
+# distutils: define_macros=CYTHON_TRACE_NOGIL=0
+import sys
+import re
+
+from cpython cimport PyString_Check, PyUnicode_Check
+
+from libc.stdlib cimport free
+
+cimport cython
+from cython cimport Py_ssize_t
+
+
+from datetime import datetime
+import time
+
+import numpy as np
+cimport numpy as np
+from numpy cimport int64_t, ndarray
+np.import_array()
+
+# Avoid import from outside _libs
```

Comment:
separately (:>), consider add ing a compat for .pyx which *can* be imported (could also simply be a .pyx that is included) to provide things like this

Reply:
Good idea.

Reply:
add this to the Todo list as well